message/message.bal

```diff
@@ -0,0 +1,22 @@
+import ballerina/http;
+
+type SurveyUpdateRequest record {
+    string title;
+    string from_template_id;
+    boolean footer;
+    string folder_id;
+    int theme_id;
```

Comment:
Isn't it a best practice in Ballerina to use camel case for variable names?

Reply:
yes, since I mock this surveyMonkey API I have used their field names.